gh-codeql

```diff
@@ -11,8 +12,8 @@ error() {
 }
 
 rootdir="$(dirname "$0")"
-channel="$(gh config get extensions.codeql.channel)"
-version="$(gh config get extensions.codeql.version)"
+channel="$(gh config get extensions.codeql.channel 2> /dev/null)" || :
```

Comment:
Minor, do only if it's easy: Should we turn this into a helper function that accepts the config key? Then we can consistently use it everywhere and not have to remember to use the same pattern.

Reply:
Unfortunately, turning it into a helper function isn't particularly easy. The only way to return a value from a function bash AIUI is to either set some global variable with the return value (which is ugly) or output on stdout (which the user would see and may be confused by). Neither of these solutions is great, so for a somewhat simple pattern like this I think it's easier to just repeat it a couple of time.